views/beer-details.hbs

```diff
@@ -0,0 +1,9 @@
+<div class="beer-container">
+    {{> beerCard beer}}
+    <ul>
+        {{#each food_pairing}}
+        <li>{{this}}</li>
+        {{/each}}
+    </ul>
+    <p>{{brewer_tips}}</p>
```

Comment:
Since you're passing the information to this view as "beer", you should wrap it all up in a `{{#with beer}}`.

As a consequence, you won't pass "beer" to the partial, but rather "this" - since it will be within the "with" statement.